outline.md

```diff
@@ -0,0 +1,55 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
```

Comment:
It's fine to say that this experience was had at Nike.  Just to present with clarity I wanted the attention to be on what we were learning about inner source rather than the company that I'm working at.  I assume that Nike is going to be attached to my name as a presenter so anyone who's curious could look at it.  What do you think?

Reply:
I can relate to this point of view very well: You do not want to give a mere experience report but whant to point out that your findings can translate to others.

However, the nice thing with a company name is that it transports a lot of context with few words: When I say Nike, words like "young", "enterprise software", "end-consumer products" pop up in my mind. When I say Siemens, other attributes pop up.

Reply:
`This presentation shares principles, practical anecdotes, and relatable examples on this situation gleaned from experience at Nike`